src/main/java/io/fabric8/elasticsearch/cloud/kubernetes/KubernetesModule.java

```diff
@@ -20,6 +20,7 @@
 import org.elasticsearch.common.settings.Settings;
```

Comment:
You can get rid of this file now since you remove its reference [here](https://github.com/fabric8io/elasticsearch-cloud-kubernetes/pull/58/files#diff-ca7096698946c72f3c380a746add6f40L24)